amf-shapes/shared/src/main/scala/amf/shapes/client/platform/model/document/JsonLDInstanceDocument.scala

```diff
@@ -0,0 +1,19 @@
+package amf.shapes.client.platform.model.document
+import amf.core.client.platform.model.document.BaseUnit
+import amf.shapes.client.scala.model.document.{JsonLDInstanceDocument => InternalJsonLDInstanceDocument}
+import amf.shapes.client.scala.model.domain.jsonldinstance.JsonLDElement
+import amf.shapes.internal.convert.ShapeClientConverters._
+
+import scala.scalajs.js.annotation.{JSExport, JSExportAll, JSExportTopLevel}
+@JSExportAll
+class JsonLDInstanceDocument(override private [amf] val _internal: InternalJsonLDInstanceDocument) extends BaseUnit {
+
+  @JSExportTopLevel("JsonLDInstanceDocument")
+  def this()= this(InternalJsonLDInstanceDocument())
+
+  def WithEncodes(encodes: ClientList[JsonLDElement]): this.type = {
+    _internal.withEncodes(encodes).asClient
+    this
+  }
+  def encodes: ClientList[JsonLDElement] = _internal.encodes
```

Comment:
You need to convert this to a `ClientList` since `_internal.encodes` will return an "Internal List". You will do this with `.asClient`